_assets/js/search.js

```diff
@@ -116,14 +116,16 @@
 
   const autocompleteContainer = document.querySelector('.autocomplete_container');
 
-  const highlight = (text) => {
-    return text.replace(/\uE000/g, '<strong>').replace(/\uE001/g, '</strong>');
```

Comment:
This original highlight appears to be relevant for search.gov results (handled by `search/index.html`), but not those coming from the `accessible-autocomplete` library.